elixir/apps/web/lib/web/live/policies/new.ex

```diff
@@ -92,7 +92,7 @@ defmodule Web.Policies.New do
                   field={@form[:description]}
                   label="Description"
                   type="textarea"
-                  placeholder="Optionally, enter a reason for creating a policy here."
+                  placeholder="Why are you creating this policy?"
```

Comment:
Not married to this placeholder.

```suggestion
                  placeholder="Enter an optional reason for creating this policy here."
```

Reply:
We don't have any visual way to show that some field is optional so I'd like to keep "Optionally" or something along those lines for all optional fields.